Add a note that explains the limitations of the schema conversion (#201)
pulp/pulp_container: docs/workflows/host.rst

     https://docs.docker.com/get-started/

.. note::

>     Having a container client which cannot handle requested manifests in the new format (schema 2),

Reviewer: ```suggestion
    When using a container client that cannot handle requested manifests in the new format (schema 2),
```
